packages/types/src/index.ts

```diff
@@ -473,10 +486,11 @@ export type Stop = {
   dist?: number;
   geometries?: { geoJson?: GeoJSON.Polygon };
   id: string;
```

Comment:
Consider adding a type alias per https://github.com/opentripplanner/otp-ui/pull/655/files#r1387985351 if we are to avoid breaking compatibility in a whole bunch of packages in #655.

Reply:
Very good suggestion, thank you